app/views/layouts/application.html.erb

```diff
@@ -126,7 +126,8 @@
           <%= govuk_notification_banner(
             title_text: "Success",
             success: true, title_heading_level: 3,
-            title_id: "flash-notice"
+            title_id: "flash-notice",
+            role: "alert",
```

Comment:
Assuming this behaviour is desired for all notifications - the added line role: "alert" makes sure to shift focus to the notification banner on page load. Not quite works this way with the tabs and anchors on this page but should do on other pages in the app without.